test/helpers/increaseTime.js

```diff
@@ -29,7 +29,7 @@ export default function increaseTime(duration) {
 }
 
 // sets time to given timestamp based on current block time
-export function setTimeTo(timestamp) {
+export async function setTimeTo(timestamp) {
```

Comment:
Yeah, let's agree on adding async keyword to `increaseTime`. 

Redoing it for sync calls is also a great idea but i don't want to do any more refactoring on this branch. Let's discuss improving things like this tmrw.